src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php

```diff
@@ -352,4 +355,15 @@ private function updateEntityStatusWithJiraTicketStatus(ManageEntity $entity, Is
             $entity->updateStatus(Constants::STATE_REMOVAL_REQUESTED);
         }
     }
+
+    private function updateStatus(ManageEntity $entity)
+    {
+        $excludeFromPush = $entity->getMetaData()->getCoin()->getExcludeFromPush();
+        if ($excludeFromPush === '1') {
+            $entity->updateStatus(Constants::STATE_PUBLICATION_REQUESTED);
+        }
+        if ($excludeFromPush === '0') {
```

Comment:
Consider throwing an exception when no valid excludeFromPush value is met e.g. '-1'.

Reply:
That is certainly possible, but would mess up test entities who might not have the exclude from push coin. So I either had to add catch clauses around the method call or leave as is. I opted for the latter.